examples/configs/rag/pinecone/config.py

```
@@ -18,11 +18,24 @@
 from typing import Optional
 
 import pinecone
-from langchain.chains import RetrievalQA
-from langchain.docstore.document import Document
-from langchain.embeddings.openai import OpenAIEmbeddings
-from langchain.vectorstores import Pinecone
-from langchain_core.language_models.llms import BaseLLM
+
+try:
```

Same as above, why do two nested checks rather than one and raise the Error with the packages the user's missing?